contrib/babelfishpg_tsql/src/linked_servers.c

```diff
@@ -94,6 +94,49 @@ linked_server_msg_handler(LinkedServerProcess lsproc, int error_code, int state,
 	return 0;
 }
 
+/*
+ * Helper function to remove all occurrences
+ * of a substring from a source string
+ */
+static char *
+remove_substr(char *src, const char *substr)
```

Comment:
src is already a copy string. You can remove keyword from it in place with src/dst pointers.